scripts/calendar/germany_school_holidays_generator.py

```
@@ -0,0 +1,447 @@
+    label = label.replace("\n", " ").strip()
+    label = re.sub(r"-\s+", "-", label)
+    label = re.sub(r"^\d++\s++", "", label)
+    label = re.sub(r"\s*+\([^)]*+\)", "", label)
+    label = label.replace("²", "").replace("*", "")
+    label = re.sub(r"\s++\d++$", "", label)
+    return re.sub(r"\s+", " ", label).strip()
+
+
+def _normalize_cell(cell: str) -> str:
+    cell = cell.replace("\n", " ")
+    cell = cell.replace("–", "-").replace("—", "-")
```

Ruff (0.15.9)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)